pom.xml

```diff
@@ -101,6 +101,10 @@
     <snappy-java.version>1.1.10.4</snappy-java.version>
     <test.build.data>${project.build.directory}/tmp</test.build.data>
     <wro4j-maven-plugin.version>1.7.9</wro4j-maven-plugin.version>
+    <jvm.args>
+      --add-opens=java.base/java.lang=ALL-UNNAMED
+      --add-opens=java.base/java.util=ALL-UNNAMED
+      --add-opens java.base/java.io=ALL-UNNAMED</jvm.args>
```

Comment:
nit: proper xml formatting
```
<jvm.args>
   ...
   ...
   ...
</jvm.args>
```

Reply:
can i put name as "test.jvm.args" ? any suggestions on name ? thx !

Reply:
test.jvm.args is okay, and it's done, thanks